tool/settings.re

```diff
@@ -99,6 +99,8 @@ struct Settings {
     player_color_red: float,
     player_color_green: float,
     player_color_blue: float,
+    flying_up_down_vel: float,
+    flying_forward_backward_vel: float,
```

Comment:
I don't think shortening `velocity` to `vel` saves much typing here, but definitely hinders understanding the code.